src/utils/iced.rs

```diff
@@ -257,7 +258,7 @@ impl<P: Program + Send + 'static> IcedElement<P> {
             pending_update: None,
             size,
             cursor_pos: None,
-            theme: Theme::dark(), // TODO
+            theme, // TODO
```

Comment:
```suggestion
            theme,
```

(This is what that `TODO` was about)